src/hooks/useCollectionColumns.ts

```diff
@@ -0,0 +1,27 @@
+import { useFragment } from 'react-relay';
+import { graphql } from 'relay-runtime';
+import { useMemo } from 'react';
+import { DEFAULT_COLUMNS } from 'constants/layout';
+import { isValidColumns } from 'scenes/UserGalleryPage/UserGalleryCollection';
+import { useCollectionColumnsFragment$key } from '__generated__/useCollectionColumnsFragment.graphql';
+
+export const useCollectionColumns = (collectionRef: useCollectionColumnsFragment$key | null) => {
```

Comment:
any reason this hook is using a fragment, rather than being a static func where the parent passes in the collection layout columns?